omnigan/blocks.py

```diff
@@ -293,6 +293,42 @@ def __str__(self):
         return strings.basedecoder(self)
 
 
+class DepthDecoder(nn.Module):
+    """#Depth decoder based on depth auxiliary task in DADA paper
+
+    """
+    def __init__(self, opts):
+        super().__init__()
+        self.relu = nn.ReLU(inplace = True)
+        self.enc4_1 = nn.Conv2d(2048, 512, kernel_size=1, stride=1, padding=0, bias=True)
+        self.enc4_2 = nn.Conv2d(512, 512, kernel_size=3, stride=1, padding=1, bias=True)
+        self.enc4_3 = nn.Conv2d(512, 128, kernel_size=1, stride=1, padding=0, bias=True)
+        self.enc4_1.weight.data.normal_(0, 0.01)
```

Comment:
there's an automatic init function run in get_generator. Either you remove things here or add an if statement there so that things are not re-initialized

Reply:
I changed that, please check that its ok, things are running